Sources/NIO/IO.swift

```diff
@@ -28,16 +32,47 @@ public struct IOError: Swift.Error {
         return .reason(self.failureDescription)
     }
 
+    private enum Error {
+#if os(Windows)
+    case WindowsError(DWORD)
+    case WinSockError(CInt)
+#endif
+    case errno(CInt)
```

Comment:
Nit: we indent enum `case` declarations.

Reply:
Re-indented.